js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jupyter-fs",
-  "version": "1.0.1",
+  "version": "1.1.0rc0",
```

Comment:
I don't think this is a valid semver string for npm packages, but that is OK if we're not publishing it?

Reply:
Yeah just pypi